src/symbols_macos.cpp

```diff
@@ -50,6 +50,17 @@ class MachOParser {
         }
     }
 
+    const section_64* findStubSection(const segment_command_64* sc) {
```

Comment:
Better to make it a general purpose `findSection` function that accepts `name` as an argument.
This will be in line with ElfParser on Linux, and will allow reuse for other sections in the future.